pkg/scheduler/internal/cache/cache.go

```diff
@@ -47,6 +47,15 @@ func New(ttl time.Duration, stop <-chan struct{}) Cache {
 	return cache
 }
 
+// nodeInfoListItem holds a NodeInfo pointer and acts as an item in a doubly
+// linked list. When a NodeInfo is updated, it goes to the head of the list.
+// The items closer to the head are the most recently updated items.
+type nodeInfoListItem struct {
+	info *schedulernodeinfo.NodeInfo
```

Comment:
If we're going to shorten the variable name for type `NodeInfo` I'd rather see `node` than `info`. After all, info is short for information, and don't all variables carry information? I am not worried that someone will get confused between `NodeInfo` and `v1.Node` in this context.

Reply:
The reason that I went with `info` is that the map structure that uses this struct is called `nodes`. So, with this field called info, we have code like `node.info` and `n.info`. If this is called `node`, then we would have `node.node` and `n.node` which would look a bit odd.